File: PMUG02_16_2021_Project/End_Java/app/src/main/java/com/neudesic/pmugfebappend/database/Converter.java
package com.neudesic.pmugfebappend.database;

import androidx.room.TypeConverter;

import org.joda.time.LocalDate;


public class Converter {
    @TypeConverter
    public LocalDate fromLongToLocalDate(Long date) {
        return new LocalDate(date);
    }

    @TypeConverter
    public long toLongFromLocalDate(LocalDate localDate) {
        return localDate.toDate().getTime();
    }
}

File: PMUG02_16_2021_Project/End_Kotlin/app/src/main/java/com/neudesic/pmug02_16_2021_kotlin/database/Converter.kt
package com.neudesic.pmug02_16_2021_kotlin.database

import androidx.room.TypeConverter
import org.joda.time.LocalDate

class Converter {
    @TypeConverter
    fun fromLongToLocalDate(long: Long) : LocalDate {
        return LocalDate(long)
    }
    @TypeConverter
    fun toLongFromLocalDate(localDate: LocalDate) : Long {
        return localDate.toDate().time
    }
}